include/jemalloc/internal/tcache_types.h

```diff
@@ -47,8 +47,8 @@ typedef struct tcaches_s tcaches_t;
 #define TCACHE_GC_INCR							\
     ((TCACHE_GC_SWEEP / SC_NBINS) + ((TCACHE_GC_SWEEP / SC_NBINS == 0) ? 0 : 1))
 
-/* Used in TSD static initializer only. Real init in tcache_data_init(). */
-#define TCACHE_ZERO_INITIALIZER {0}
```

Comment:
Any reason to remove the 0? I think as long as it doesn't hurt, we should keep it.

Reply:
The `0` was previously for initializing the `prof_accumbytes` field in `tcache`, or more precisely, `{0}` forms a partial initialization list for the entire `tcache` structure. Now after `prof_accumbytes` is removed from the `tcache`, the code won't build if I leave the `{0}` initializer as is, since now the first field is the GC ticker. So I removed `0` and just supplied an empty `{}` initializer and the code is able to build again.

However, interestingly, I just found in a CI run that MSVC doesn't like my empty initializer. So I'm now changing the initializer to `{{0, 0}}` and see if MSVC likes it... (The `{0, 0}` in the middle is for initializing the two internal fields of the GC ticker.)

Reply:
empty initializers may be disallowed.  `{{0}}` may work as well.